src/Form/TypeExtension/PhoneNumberTypeExtension.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace AppBundle\Form\TypeExtension;
+
+use libphonenumber\PhoneNumberUtil;
+use libphonenumber\PhoneNumber;
+use Misd\PhoneNumberBundle\Form\Type\PhoneNumberType;
+use Symfony\Component\Form\AbstractTypeExtension;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+
+class PhoneNumberTypeExtension extends AbstractTypeExtension
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder->addEventListener(FormEvents::PRE_SET_DATA, [$this, 'setDefaultRegion']);
+    }
+
+    public function getExtendedType()
+    {
+        return PhoneNumberType::class;
+    }
+
+    public function setDefaultRegion(FormEvent $event)
+    {
+        $form = $event->getForm();
+        $defaultRegion = $form->getConfig()->getOption('default_region');
+
+        if (!$event->getData() && PhoneNumberUtil::UNKNOWN_REGION !== $defaultRegion) {
+            $event->setData((new PhoneNumber())->setCountryCode(33)->setNationalNumber(''));
```

Comment:
Hmm shounld't we do like https://github.com/EnMarche/en-marche.fr/pull/2111/files#diff-d4d5073bfd6cedfc2c729bbb3fe53b99R141?

Reply:
I'll change this in the PR which uses the local to set default phone number country.